lib/index.js

```diff
@@ -26,91 +7,90 @@ const Models = require('./models');
+    if (options.models) {
+        const files = await Models.getFiles(options.models);
+        let models = await Models.load(files, options.sequelize.import.bind(options.sequelize));
         models = await Models.applyRelations(models);
 
-        if (opts.sync) {
-            await opts.sequelize.sync({ force: opts.forceSync });
+        if (options.sync) {
+            await options.sequelize.sync({ force: options.forceSync });
         }
 
-        db = new DB(opts.sequelize, models);
+        db = new DB(options.sequelize, models);
     } else {
-        db = new DB(opts.sequelize, []);
+        db = new DB(options.sequelize, []);
     }
 
-    if (opts.onConnect) {
-        let onConnect = opts.onConnect(db);
-        // if is a promise, wait until it finishes
+    if (options.onConnect) {
+        let onConnect = options.onConnect(db);
+
         if (onConnect && typeof onConnect.then === 'function') {
             await onConnect;
         }
     }
 
     return db;
-};
-
-module.exports = {
-    pkg: Pkg,
-    once: true,
-    async register(server, options) {
-        assert(options, 'Missing hapi-sequelizejs plugin options');
-
-        if (!Array.isArray(options)) options = [options];
-
-        await Joi.validate(options, Schema.options);
-
-        const getDb = request => {
-            return function getDb(name) {
-                if (!name) {
-                    const key = Object.keys(request.server.plugins[Pkg.name]).shift();
-                    return request.server.plugins[Pkg.name][key];
-                } else if (!request.server.plugins[Pkg.name].hasOwnProperty(name)) {
-                    throw new Error(`hapi-sequelizejs cannot find the ${name} database instance`);
-                }
-
-                return request.server.plugins[Pkg.name][name];
-            };
-        };
-
-        server.decorate('request', 'getDb', getDb, {
-            apply: true,
-        });
-
-        server.events.on('stop', async () => {
-            const dbNames = options.map(option => option.name);
-            const pluginContent = server.plugins['hapi-sequelizejs'];
-
-            await Promise.all(dbNames.map(dbName => pluginContent[dbName].sequelize.close()));
-        });
-
-        const configured = options.reduce(
-            (acc, opts) => [
-                ...acc,
-                internals.configure(opts).then(db => {
-                    server.expose(opts.name, db);
-                    return db;
-                }),
-            ],
-            [],
-        );
+}
+
+function getDb(request) {
+    return function getDb(name) {
+        if (!name) {
+            const key = Object.keys(request.server.plugins[Pkg.name]).shift();
+            return request.server.plugins[Pkg.name][key];
+        } else if (!request.server.plugins[Pkg.name].hasOwnProperty(name)) {
+            throw new Error(`hapi-sequelize cannot find the ${name} database instance`);
```

Comment:
It should be `hapi-sequelizejs` because it is the plugin name

Reply:
Fixed in 9f8633e6dd6f4023d67cb72e64e487a7e5cb4566